next.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
           '/organizations/:organization_id(\\d{1,})/trees/:tree_id(\\d{1,})',
         destination: '/trees/:tree_id(\\d{1,})',
       },
+      {
+        source: '/map/:map_name(\\d{1,})',
+        destination: '/?map=:map_name',
+      },
```

Comment:
@okdv I am trying to solve this, but not finished yet

Reply:
@dadiorchen hmm that seems to be working okay for me. its a funky to catch since the destination doesnt exist yet though. hows it failing on your side?

Reply:
Don't worry about this, it's in progress, we can back to this when the feature of customizing the map is ready, that's big and challenging work ahead.